config/secret

```diff
@@ -3,3 +3,8 @@
 import Config
 
 config :notesclub, :github_api_key, "GITHUB_API_KEY"
+
+config :ueberauth, Ueberauth.Strategy.Github.OAuth,
+  client_id: "GITHUB_OAUTH_CLIENT_ID",
+  client_secret: "GITHUB_OAUTH_CLIENT_SECRET"
+  
```

Comment:
The last line has two spaces. GitHub complains unless there is an empty line at the end.
```suggestion

```

Reply:
7ebb67cecbdfe064f4db480cf256dabe5af8a84b